### ISSUE TITLE
The Editbox shows a problem on Huawei mate9.

### ISSUE THREAD

**Issue by 190420456** [cocos2d/cocos2d-x#19103]
When the keyboard pops up, the screen is stretched and the resolution is wrong.

**Comment by gestern** [comment]
@190420456 You are right and you should yourself determine the behavior of your game in this case, since there are two more cases:

1. The physical size of the keyboard is determined by the choice of the type of input field (for example, for the types "email" or "only numbers" - the dimensions will be different (and unknown to you in advance and determined by the operating system for end-user device);
2. The size of the keyboard also depends on the mode in which the device is in portrait or landscape modes, so the size of the keyboard will be very different.

You have to listen `IMEDelegate` and `EditBoxDelegate` events to figure out 1 case. And have to subscribe with `AppDelegate::applicationScreenSizeChanged()` if 2nd case.

> **IMPORTANT** Do not forget that there are mobile devices that have a built-in keyboard.
